File: src/clients/java/src/java/src/com/tigerbeetle/CreateAccountsResult.java
package com.tigerbeetle;

import java.nio.ByteBuffer;

public final class CreateAccountsResult {

    static final class Struct {
        public static final int SIZE = 8;
    }

    public final int index;
    public final CreateAccountResult result;

    CreateAccountsResult(ByteBuffer ptr) {
        index = ptr.getInt();
        result = CreateAccountResult.fromValue(ptr.getInt());
    }
}

File: src/clients/dotnet/src/TigerBeetle/CreateAccountsResult.cs
using System.Runtime.InteropServices;

namespace TigerBeetle
{
    [StructLayout(LayoutKind.Sequential, Size = 8)]
    public struct CreateAccountsResult
    {
        #region Fields

        private readonly int index;

        private readonly CreateAccountResult result;

        #endregion Fields

        #region Constructor

        internal CreateAccountsResult(int index, CreateAccountResult result)
        {
            this.index = index;
            this.result = result;
        }

        #endregion Constructor

        #region Properties

        public int Index => index;

        public CreateAccountResult Result => result;

        #endregion Properties
    }
}
